packages/core/src/types.ts

```diff
@@ -230,19 +230,44 @@ export type RecursiveDesignTokenDefinition = {
   [key: string]: string | RecursiveDesignTokenDefinition;
 };
 
+/** not yet implemented, don't use */
+type DesignBorderTokenStyle = 'solid' | 'dashed' | 'dotted';
+/** not yet implemented, don't use */
```

Comment:
Maybe we can provide something more formal since this is customer facing - normally I wouldn't comment if it was internal code but since devs outside Contentful will see this, would be worth putting something that would formally present the message like

```
/** We are currently in the process of implementing this feature. Do not use this field in your designTokenDefinition. **/
```